### PR TITLE
Ecos's card

### PR DESCRIPTION
**Pull request by eco-akram**
Added a new card :)

### CODE REVIEW

**Review by Amazening — CHANGES_REQUESTED** [review]
Hey there, looks pretty good!

Just two things to note before I approve your request

- You haven't provided a link to _your_ specific GitHub handle, only the main GitHub website. Give that a quick fix.
- Your code is indented to the right - try and align it with the other cards

Other than that your code seems to look good! Just make those changes and I'll happily merge your pull request. Good luck!

**Comment by eco-akram** [comment]
Hi!
I have resolved the issue.

File: index.html [hunk 1]
```diff
@@ -121,7 +121,32 @@ <h2>A project for first-time contributions</h2>
...
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/eco-akram" target="_blank">eco.akram</a>
+           </p>
+          <p class="about">Hey, I'm an aspiring IT student from Lithuania, pretty exited to make my first contribution.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="freecodecamp.org" target="_blank" title="First Resource">FreeCodeCamp</a>
```
Comment (TenTraicion) [inline index.html:137]:
```suggestion
                <a href="https://freecodecamp.org" target="_blank" title="First Resource">FreeCodeCamp</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@eco-akram, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**